### ISSUE TITLE
Missing Error Prone dependency causes failures with some checks enabled

### ISSUE THREAD

**Issue by ekuefler** [bazelbuild/bazel#1008]
If I try to build something that has a string equality error with `--javacopt="-Xep:StringEquality:ERROR"` turned on, I get an error containing the following:

```
com.google.common.util.concurrent.ExecutionError: java.lang.NoClassDefFoundError: org/checkerframework/javacutil/AnnotationProvider
```

Looks like Bazel is missing a transitive dep of Error Prone. (This was run on Bazel 0.1.4 just in case it's been added since then)

**Comment by kchodorow** [comment]
Still an issue at head.

**Comment by ulfjack** [comment]
@eaftan @cushon

**Comment by cushon** [comment]
07caf827030ec275ad9845f942c14d8ed5fdd4e1 wasn't a complete fix, I missed this: http://mvnrepository.com/artifact/org.checkerframework/javacutil/1.8.10

**Comment by cushon** [comment]
I think @damienmg fixed this with f6c24decc9cea46bfba41efb07f11b1ce7415cd0.
